.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,369 @@
+`documentation` | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aw7sst%2FMorseRunner+label%3Adocumentation) | Related to any type of documentation.
+`enhancement` | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aw7sst%2FMorseRunner+label%3Aenhancement) | New feature or requests, including contest nominations.
+`feedback` | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aw7sst%2FMorseRunner+label%3Afeedback) | General feedback; more than bug reports and feature requests.
+`question` | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aw7sst%2FMorseRunner+label%3Aquestion) | Further information is requested. Questions that are more than bug reports or feature requests.
+`training` | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aw7sst%2FMorseRunner+label%3Atraining) | Related to any type of training or presentation material (e.g. club presentation).
+
+#### Issue Attribute
+Label name | Search | Description
+-----------|--------|------------
+`good first issue` | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aw7sst%2FMorseRunner+label%3A"good+first+issue") | Less complex issues which would be good first issues to work on for users who wanted to contribute to MorseRunner.
+`beginner` | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aw7sst%2FMorseRunner+label%3Abeginner) | Less complex issues which would be good first issues to work on for users who wanted to contribute to MorseRunner.
+
+#### Issue State
+
+Label name | Search | Description
+-----------|--------|------------
+`active` | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aw7sst%2FMorseRunner+label%3Aactive) | Issues being actively worked on by a community member.
+`blocked` | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aw7sst%2FMorseRunner+label%3Ablocked) | Issues blocked on other issues.
+`confirmed` | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aw7sst%2FMorseRunner+label%3Aconfirmed) | Issues have been confirmed and ready to be fixed/implemented.
+`duplicate` | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aw7sst%2FMorseRunner+label%3Aduplicate) | This issue or pull request already exists.
+`help-wanted` | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aw7sst%2FMorseRunner+label%3Ahelp-wanted) | Extra attention is needed. The MorseRunner team would appreciate help from the community in resolving these issues.
+`invalid` | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aw7sst%2FMorseRunner+label%3Ainvalid) | Issues which aren't valid (e.g. user errors).
+`more-information-needed` | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aw7sst%2FMorseRunner+label%3Amore-information-needed) | Likely bugs, but haven't been reliably reproduced.
```

Comment:
Decide if a hyphen should be used or a space. Look at fix applied to `good first issue`.

Reply:
Agreed to go with the space to follow convention of other labels in GitHub.